package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "compass.js",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

Comment:
Let's keep version changes out of the feature branch. We could decide to release several features in 1 version.

Reply:
check